src/DIPS.Xamarin.UI/Controls/RadioButtonGroup/RadioButtonGroup.xaml.cs

```diff
@@ -97,7 +97,26 @@ public partial class RadioButtonGroup : ContentView, IHandleRadioButtons
             Color.Black,
             propertyChanged: TextColorPropertyChanged);
 
-        private readonly IList<RadioButton> m_radioButtons = new List<RadioButton>();
+        private readonly IList<RadioButton> m_radioButtons = new List<RadioButton>();
+
+        /// <summary>
+        ///     Padding of the radiobutton group.

```

Comment:
It's not the padding of the radio button group, but for each radiobutton. The radio button group padding would indicate to the consumer that they can pad the entire outer "container" and not each item.